dashboard/templates/admin.ejs

```diff
@@ -1,44 +1,32 @@
 <%- include('blocks/header', {bot, user, path}) %>
```

Comment:
I do not know what you have done to this page, as I cannot access it **at all** when I launch the bot with these changes. Including going directly to the URL.

Reply:
I will add a live dashboard link

Reply:
found the issue